modules/claims_api/lib/claims_api/poa_pdf_constructor/organization.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require 'claims_api/poa_pdf_constructor/base'
+require 'claims_api/poa_pdf_constructor/signature'
+
+module ClaimsApi
+  module PoaPdfConstructor
+    class Organization < ClaimsApi::PoaPdfConstructor::Base
+      protected
+
+      def page1_template_path
+        Rails.root.join('modules', 'claims_api', 'config', 'pdf_templates', '21-22', '1.pdf')
+      end
+
+      def page2_template_path
+        Rails.root.join('modules', 'claims_api', 'config', 'pdf_templates', '21-22', '2.pdf')
+      end
+
+      def page1_signatures(_signatures)
+        []
+      end
+
+      def page2_signatures(signatures)
+        [
+          ClaimsApi::PoaPdfConstructor::Signature.new(data: signatures['veteran'], x: 35, y: 263),
+          ClaimsApi::PoaPdfConstructor::Signature.new(data: signatures['representative'], x: 35, y: 216)
+        ]
+      end
+
+      # rubocop:disable Layout/LineLength
+      def page2_options(data)
+        base_form = 'F[0].Page_2[0]'
+        {
+          "#{base_form}.SocialSecurityNumber_FirstThreeNumbers[0]": data.dig('veteran', 'ssn')[0..2],
+          "#{base_form}.SocialSecurityNumber_SecondTwoNumbers[0]": data.dig('veteran', 'ssn')[3..4],
+          "#{base_form}.SocialSecurityNumber_LastFourNumbers[0]": data.dig('veteran', 'ssn')[5..9],
```

Comment:
I'm prob just missing something, but can the 3rd segment of an SSN have 5 digits?
I'm just thinking that the range `5..9` is looking to pull out 5 digits and AFAIK there's only 4 digits in the 3rd segment of an SSN.

Reply:
this is something that was copied from what existed so this had me wondering if i was missing something and this was trying to protect against someone with 10 digits in their ssn or something haha.

i tried it out in console and it looks like it just gets you the last 4 even though the last 5 were asked for.

but, i tried to look into it and make sure everyone always only has 9 digits in their ssn, and it sounds like that is the case, so fixed it here:
https://github.com/department-of-veterans-affairs/vets-api/pull/6812/commits/1c17d2e63fb4ef145a96eb92ad9890561941eea7